docs/examples/create_attribute.rst

```diff
@@ -189,11 +187,8 @@ You can query the attribute to get any of the values, and even add an observer a
 
 .. note::
 
-    It is not possible to reliably use this approach to create independent modules because:
-
-    * Only one MAVLink callback can be set at a time in the running program (``set_mavlink_callback()``) 
-    * There is no reliable way to import modules without adding them properly to the Python environment.
-
+    It is not possible to reliably use this approach to create independent modules because only one 
```

Comment:
This is now incorrect, but we will need to document / spec out on_message first.